devolutions-gateway/src/interceptor/pcap_recording.rs

```diff
@@ -20,15 +21,17 @@ pub struct PcapRecordingInterceptor {
     recorder: Arc<Mutex<Option<Recorder>>>,
     condition_timeout: Arc<(Mutex<RecordingState>, Condvar)>,
     handle: Arc<Mutex<Option<thread::JoinHandle<()>>>>,
+    file_name: Arc<Mutex<String>>,
```

Comment:
Maybe document the reason why it's not using async mutexes at this place in a comment

Reply:
done